### PR TITLE
#167164935 setup travis

### PR DESCRIPTION
**Pull request by Rythae**
#### Pivotal Tracker Story

[#167164935](https://www.pivotaltracker.com/story/show/167164935)

#### What does this PR do?
 set up travis

#### Summary of Task
Screenshot 2019-08-28 at 5 23 11 PM

- installed jest and enzyme
- add test script
- configure travis yml

 ### How can this be tested?
- git clone branch `chore/167164935/update-travis`
- install dependencies
- run npm install
- start dev server npm run dev

### CODE REVIEW

**Review by OvieMudi — APPROVED** [review]
LGTM

File: src/config/assetsTransformer.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const path = require('path');
```
Comment (Nkemjiks) [inline src/config/assetsTransformer.js:1]: What is this file for?

Comment (Rythae) [reply src/config/assetsTransformer.js:1]: It used for parsing CSS and SCSS files. Please refer to line 71 of our package.json file where we used it.
And it helped sort out our parsing errors with SCSS

**Review by Nkemjiks — APPROVED** [review]
LGTM

**Pull request merged**